activities.el

```diff
@@ -495,6 +530,59 @@ accordingly."
       (setf activities-mode-timer nil))
     (remove-hook 'kill-emacs-hook #'activities-mode--killing-emacs)))
 
+(require 'easymenu)
+
+(setq activities-menu-item-resume
+      '("Resume..."
+        :help "Resume an existing activity"
+        :filter (lambda (&optional _)
+                  (let ((current-activity-name
+                         (when-let ((current-activity (activities-current)))
+                           (activities-activity-name current-activity))))
+                    (mapcar (lambda (act)
+                              (vector act `(activities-resume (activities-named ,act))
+                                      :style 'radio
+                                      :selected (equal current-activity-name act)
+                                      ))
+                            (activities-names))))
+        ))
+
+(easy-menu-define activities-menu activities-mode-map
+  "Activities Menu"
+  `("Activities" :visible activities-mode
+    ["New" activities-new
+     :help "Create a new, empty activity"]
+    ["Define" activities-define
+     :help "Create a new activity using the current frame/tab"]
+    ["Resume" activities-resume
+     :help "Resume an existing activity"]
+    ,activities-menu-item-resume
+    ["Revert" activities-revert
+     :help "Revert the current activity to its original state"]
+    ["Suspend" activities-suspend
+     :help "Suspend the specified live activity and save its current state"]
+    ["Kill" activities-kill
+     :help "Revert and suspend the specified live activity"]
+    ["Switch" activities-switch
+     :help "Focus on the specified live activity"]
+    ["Switch Buffer" activities-switch-buffer
+     :help "Focus on the specified buffer from a live activity (activities-tabs-mode only)"]
+    ["List" activities-list
+     :help "Show the master list of known activities"]
+    ["Rename" activities-rename
+     :help "Rename the specified activity"]
+    ["Discard" activities-discard
+     :help "Discard the specified activity (this is undoable)"]
+    ["Save All" activities-save-all
+     :help "Save all live activities to disk"]
+    ))
```

Comment:
It would be helpful to roughly organize these entries by purpose, with some separators between major types.

Also, hanging parens.  :)

Reply:
Done.